src/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>To Do List</title>
+</head>
+<body>
+    <br>
+    <div id="data">
+        <h1> to do list</h1>
+    </div>
```

Comment:
- [OPTIONAL ]  It would be great if you had added the input form as well, so that the app looks geat as it in provided video 
![image](https://user-images.githubusercontent.com/43877689/178940988-ef2b0609-07fa-4b68-90d8-01148ecf1af2.png)